Tests/Fixtures/DBAL/Types/AbstractParentType.php

```diff
@@ -19,9 +19,9 @@
  *
  * @author Arturs Vonda <github@artursvonda.lv>
  *
- * @template T of int|string
+ * @template TValue of int|string
  *
- * @extends AbstractEnumType<T>
+ * @extends AbstractEnumType<TValue, string>
```

Comment:
This example deliberately sets the readable type to `string` to show that children will "inherit" this but can still modify the value type.